README.md

```diff
@@ -174,6 +174,24 @@ string, default: `my-cluster`
 
 Name of the cluster.
 
+#### `ha_cluster_cluster_properties`
+
+structure, default: no properties
+
+```yaml
+ha_cluster_cluster_properties:
+  - attrs:
```

Comment:
So these are the attributes that are global, or cluster wide, or that apply to every host in the cluster?

What other elements might be in this array?
```yaml

      
    ha_cluster_cluster_properties:
      - attrs:
          - name: property1_name
            value: property1_value
          ...
      - somethingelse?
          - name: property1_name
            value: property1_value
```
?



    
    
      
          - name: property2_name




    
    
      
    <span class="blob-code-inner blob-code-marker" data-code-marker="+"><span class="pl-s1">        <span class="pl-ent">value</span>: <span class="pl-s">property2_value</span></span></span>

Reply:
These attributes are cluster wide, they apply to the cluster itself. They are documented at https://clusterlabs.org/pacemaker/doc/en-US/Pacemaker/2.0/html/Pacemaker_Explained/s-cluster-options.html

There may be list of sets of properties. This is not supported yet, but it is a requested feature. Each set can have a rule which defines when the set applies and an id (name). I have not designed that in the details yet, the rough idea is:
```yaml
    ha_cluster_cluster_properties:
      - attrs:
          - name: property1_name
            value: property1_value
          - name: property2_name
            value: property2_value
      - attrs:
          - name: property1_name
            value: property1_value_for_weekend
        rule: date-spec weekdays=6-7
        id: weekend-properties
```
This sets value of `property1_name` to `property1_value_for_weekend` on weekends and to `property1_value` on Monday through Friday. Value of `property2_name` is constant.

Reply:
ok - so if a set of properties has no `rule` then the properties apply at all times to all nodes - and if there is no `id` then the implicit id is `default` or `global`

Reply:
> if a set of properties has no `rule` then the properties apply at all times to all nodes

yes

> if there is no `id` then the implicit id is `default` or `global`

implicit id is `cib-bootstrap-options` - it is set by pacemaker